app/views/orders/index.html.erb

```diff
@@ -0,0 +1,80 @@
+<div class="page-header">
+  <h1><%= @seller.username %> <small><%= @seller.email %></small></h2>
+</div>
+
+<div class="row">
+  <%= link_to "View All Orders", seller_orders_path(seller_id: @seller.id), class: "btn btn-default" %>
+  <%= link_to "View Shipped Orders Only", seller_orders_path(seller_id: @seller.id, status: "complete"), class: "btn btn-default" %>
+  <%= link_to "View Paid Orders Only", seller_orders_path(seller_id: @seller.id, status: "paid"), class: "btn btn-default" %>
+  <%= link_to "View Pending Orders Only", seller_orders_path(seller_id: @seller.id, status: "pending"), class: "btn btn-default" %>
+  <%= link_to "View Canceled Orders Only", seller_orders_path(seller_id: @seller.id, status: "canceled"), class: "btn btn-default" %>
+</div>
+
+<% if @orders.length > 0 %>
+  <table class="table table-striped table-condensed">
+    <caption>Revenue by Order Status</caption>
+    <thead>
+      <tr>
+        <th>Order Status</th>
+        <th>Number of Orders</th>
+        <th>Total</th>
+      </tr>
+    </thead>
+     <tr>
```

Comment:
I wonder if this whole section could be cleaned up with an array that contained the statuses since the rest of the markup is the same